controllers/controller.js

```diff
@@ -42,3 +44,32 @@ exports.getCommentsByReviewId = (req, res, next) => {
       next(err);
     });
 };
+
+exports.postCommentByReviewId = (req, res, next) => {
+  const { review_id } = req.params;
+  const newReview = {
+    body: req.body.body,
+    votes: 0,
+    author: req.body.username,
+    review_id: review_id,
+    created_at: new Date(1610964545411),}
```

Comment:
You don't have to specify the date here, sql should assign the date dynamically when the new comment is posted